plugins/woocommerce/client/blocks/assets/js/blocks/checkout/styles/style.scss

```diff
@@ -1,3 +1,37 @@
+@mixin skeleton-animation {
```

Comment:
This is a temporary improvement of the Skeleton we show until the block loads. I'll reiterate and look for better ways to improve it in a different PR